packages/botfuel-dialog/src/adapters/adapter.js

```diff
@@ -75,7 +76,7 @@ class Adapter {
     await this.middlewareManager.out(context, async () => {
       for (const botMessage of botMessages) {
         // eslint-disable-next-line no-await-in-loop
-        await this.sendMessage(botMessage);
+        await this.sendMessage(this.addProperties(botMessage));
```

Comment:
I would have prefered extendMessage rather than addProperties but it is subjective